pkg/agent/qrm-plugins/memory/dynamicpolicy/policy.go

```diff
@@ -255,12 +261,22 @@ func (p *DynamicPolicy) Start() (err error) {
 	}
 	p.stopCh = make(chan struct{})
 
+	p.registerControlKnobHandlerCheckRules()
 	go wait.Until(func() {
 		_ = p.emitter.StoreInt64(util.MetricNameHeartBeat, 1, metrics.MetricTypeNameRaw)
 	}, time.Second*30, p.stopCh)
+
+	general.RegisterHeartbeatCheck(memconsts.HealthCheckClearResidualState, stateCheckPeriod*healthCheckGracePeriodNums,
+		general.HealthzCheckStateNotReady, stateCheckPeriod*healthCheckGracePeriodNums)
```

Comment:
is it possilble to replace "go wait.Until(xxx)" with "periodicalhandler.RegisterPeriodicalHandler" and itegrate  RegisterHeartbeatCheck into RegisterPeriodicalHandler ?

Reply:
get, I'll try to refine it.